boards/riscv/it8xxx2_evb/doc/index.rst

```diff
@@ -73,6 +75,12 @@ Other hardware features are not currently supported by Zephyr.
 The default configuration can be found in the
 :zephyr_file:`boards/riscv/it8xxx2_evb/it8xxx2_evb_defconfig` Kconfig file.
 
+Hardware reworks
+****************
+
+Before using the it8xxx2_evb, some hardware rework is needed. It is suggested
+to ask the board vendor for the rework guide.
```

Comment:
IIRC, this is were the long and detailed HW rework guide was before. Is it available somewhere on the vendor site? If so, can we link it here?

Reply:
@Dino-Li @GTLin08  is it possible to upload a rework document somewhere, eg on ITE web site?

Reply:
@hunterhu3000 sure, we will get back to you once it is ready.

Reply:
@hunterhu3000 the rework document has been uploaded here:
https://www.ite.com.tw/uploads/product_download/IT81302_MECC_Rework_Guide_0927.pdf

Reply:
@fkokosinski all the change requests are updated. Thanks to review and +1